v2/css/main.less

```diff
@@ -121,6 +121,7 @@ header[role="header"] {
 			span { color: @accent2; font-weight: 300; }
  		}
  		small { display: block; font-size: 13px; color: @bg; }
+        img {margin-top: -6px; margin-left: -20px; margin-right: 10px;}
```

Comment:
@membphis it was a CDN cache purge issue and only visible in China mainland. Alas. Just purges the CSS file again and it should work now.

BTW, we need a space after `{` and also a space before `}` in your CSS line.